locales/templates/app.pot

```diff
@@ -1556,77 +1556,617 @@ msgid ""
+msgid "Fast access to your favourite games"
+msgstr ""
+
+msgctxt "gaming_vertical_hint_body"
+msgid "Press to create a shortcut to your home screen."
+msgstr ""
+
+msgctxt "gaming_vertical_hint_button"
+msgid "OK"
+msgstr ""
+
+#. Gaming Longpress Menu
+msgctxt "gaming_vertical_menu_option_1"
+msgid "Share game"
+msgstr ""
+
+msgctxt "gaming_vertical_menu_option_2"
+msgid "Create shortcut on home screen"
+msgstr ""
+
+msgctxt "gaming_vertical_menu_option_3"
+msgid "Remove from list"
+msgstr ""
+
+#. Gaming Share Message. %1$s is the url of the game
+#, c-format
+msgctxt "gaming_vertical_share_message"
 msgid ""
-"We plan to integrate “Free Wifi-Finder” to help you find quality free "
-"internet connection nearby.\n"
-"\n"
-" Interested?"
+"I’m playing a really cool game. Check it out!\n"
+"%1$s"
+msgstr ""
+
+#. Gaming Genres
+msgctxt "gaming_vertical_genre_1"
+msgid "Recently played"
+msgstr ""
+
+msgctxt "gaming_vertical_genre_2"
+msgid "Game of the week"
+msgstr ""
+
+msgctxt "gaming_vertical_genre_3"
+msgid "My game"
+msgstr ""
+
+msgctxt "gaming_vertical_genre_4"
+msgid "Most played"
+msgstr ""
+
+msgctxt "gaming_vertical_genre_5"
+msgid "Offline games"
+msgstr ""
+
+msgctxt "gaming_vertical_genre_6"
+msgid "Strategy"
+msgstr ""
+
+msgctxt "gaming_vertical_genre_7"
+msgid "Adventure"
+msgstr ""
+
+msgctxt "gaming_vertical_genre_8"
+msgid "Action"
+msgstr ""
+
+msgctxt "gaming_vertical_genre_9"
+msgid "Arcade"
+msgstr ""
+
+msgctxt "gaming_vertical_genre_10"
+msgid "Puzzle & Logic"
+msgstr ""
+
+msgctxt "gaming_vertical_genre_11"
+msgid "Sports & Racing"
+msgstr ""
+
+msgctxt "gaming_vertical_genre_12"
+msgid "Girls"
+msgstr ""
+
+#. Home 2.0
+#. Onboarding for existing users to upgrage to Firefox Lite 2.0. %1$s will be
+#. replaced with the name of the app (e.g. Firefox Lite)
+#, c-format
+msgctxt "onboarding_2_introduction_title"
+msgid "Brand-new %1$s"
+msgstr ""
+
+msgctxt "onboarding_2_introduction_description"
+msgid ""
+"Remain Fast and Lightweight. We added smart shopping search and content "
+"to make internet browsing more fun and enjoyable!"
+msgstr ""
+
+#. Onboarding for new users to introduce Firefox Lite 2.0
+#. Introduce Content & Services
+msgctxt "onboarding_2_content_services_news_shopping_games"
+msgid ""
+"Tap to read the trendy news, find good deals or play the hottest free "
+"games to have fun!"
+msgstr ""
+
+msgctxt "onboarding_2_content_services_news_games"
+msgid "Tap to read the trendy news or play the hottest free games to have fun!"
+msgstr ""
+
+msgctxt "onboarding_2_content_services_games"
+msgid "Tap to play the hottest free games to have fun!"
+msgstr ""
+
+#. Introduce Smart Shopping Search
+msgctxt "onboarding_2_smart_shopping_search_title"
+msgid "Smart Shopping Search"
+msgstr ""
+
+msgctxt "onboarding_2_smart_shopping_search_description"
+msgid "Comparing prices across different online stores is super easy, try it now!"
+msgstr ""
+
+msgctxt "home_search_bar_text"
+msgid "Search..."
```

Comment:
This should be a correct ellipsis

Reply:
Got it, have revised to the correct ellipsis